src/emmy/expression/compile.cljc

```diff
@@ -481,8 +505,7 @@
          body     (-> (apply f args)
                       (g/simplify)
                       (v/freeze)
-                      (cse-form)
-                      (apply-numeric-ops))
+                      (cse {}))
```

Comment:
You might also give `cse` a zero-arity version that supplies this default empty options map

Reply:
done